src/menu/menu.scss

```diff
@@ -36,19 +37,28 @@
 
     > .iui-content {
       flex: 1 1 auto;
-      white-space: nowrap;
-      overflow: hidden;
-      text-overflow: ellipsis;
-      height: $iui-line-height;
+
+      .iui-menu-description {
+        font-size: $iui-font-size-small;
+        @include themed {
+          opacity: t(iui-opacity-3);
+        }
+      }
 
       + .iui-icon {
         margin-right: 0;
         margin-left: $iui-s;
       }
+
+      &,
+      > * {
```

Comment:
Not sure about `*`

Reply:
Removed this, gave every nested div a class name instead.